README.md

```diff
@@ -1,7 +1,9 @@
 ## Test262 is the official ECMAScript (ECMA-262) conformance suite.
 
+Test262 tests conformance to the continually maintained draft future ECMAScript standard found at http://tc39.github.io/ecma262/ , together with any Stage 3 or later TC39 proposals. It is maintained by Gorkem Yakin (@goyakin) with significant contributions from many in the ECMAScript community.
+
```

Comment:
> Well, Google would not, and does not intend to contract Bocoup to do any work on maintaining that data.

Ok, so what does this mean practically speaking? If the test262 authors agree here that we want to continue requiring this metadata, what will Google do? Cancel the contract? Fork the project over a trivial amount of work to support more use cases?